internal/provider/resources/resource_deployment_test.go

```diff
@@ -324,7 +325,228 @@ func TestAcc_ResourceDeploymentStandard(t *testing.T) {
 				ResourceName:            gcpKubernetesResourceVar,
 				ImportState:             true,
 				ImportStateVerify:       true,
-				ImportStateVerifyIgnore: []string{"external_ips", "oidc_issuer_url", "environment_variables.1.value"}, // environment_variables.0.value is a secret value
+				ImportStateVerifyIgnore: []string{"external_ips", "oidc_issuer_url", "scaling_status", "environment_variables.1.value"}, // environment_variables.0.value is a secret value
+			},
+		},
+	})
+}
+
+func TestAcc_ResourceDeploymentStandardScalingSpec(t *testing.T) {
+	namePrefix := utils.GenerateTestResourceName(10)
+
+	scalingSpecDeploymentName := fmt.Sprintf("%v_scaling_spec", namePrefix)
+	scalingSpecResourceVar := fmt.Sprintf("astro_deployment.%v", scalingSpecDeploymentName)
+
+	// standard aws deployment
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: astronomerprovider.TestAccProtoV6ProviderFactories,
+		PreCheck:                 func() { astronomerprovider.TestAccPreCheck(t) },
+		Steps: []resource.TestStep{
+			{
+				Config: astronomerprovider.ProviderConfig(t, true) + standardDeployment(standardDeploymentInput{
+					Name:              scalingSpecDeploymentName,
+					Description:       utils.TestResourceDescription,
+					Region:            "us-east4",
+					CloudProvider:     "GCP",
+					Executor:          "CELERY",
+					SchedulerSize:     "SMALL",
+					IsDevelopmentMode: true,
+					ScalingSpec:       `scaling_spec = {}`,
+				}),
+				ExpectError: regexp.MustCompile(`Inappropriate value for attribute "scaling_spec"`),
```

Comment:
nit: `invalid` over `inappropriate` is more accurate here

Reply:
inappropriate is what the terraform error outputs through their library